src/libs/blueprint/conduit_blueprint.hpp

```diff
@@ -64,6 +64,7 @@
 #include "conduit_blueprint_mcarray.hpp"
 #include "conduit_blueprint_mcarray_examples.hpp"
 
+#include "conduit_blueprint_zfparray.hpp"
```

Comment:
Looking at the latest CI build failure, this needs to be removed or conditionally included. If users build with ZFP and wish to use the blueprint zfp support, should they be expected to explicitly include this header, or have it automatically included with this file (conduit_blueprint.hpp)?

Reply:
i think its fine to conditionally include it here.

That reminds we that we should also add it to this list:
https://github.com/LLNL/conduit/blob/master/src/libs/blueprint/conduit_blueprint.cpp#L94

Which can help folks query want is enabled at runtime.